### PR TITLE
fix: apply rtl styles for algolia when dir is `rtl`

### LINKED ISSUE #624: Search field display is ruptured
## Description

From the following commit https://github.com/nuxt/nuxtjs.org/commit/40e83a976435eaf73faa3e79d81d64671484743b#diff-f332d6095fb5ad28b916b08cf5426070

The search field display is ruptured:
- The search icon is clung to the input field
- RTL used by default 

The `rtl.css` style is overriding the tailwindcss style for all language direction.

#### In english
Capture d’écran 2020-08-31 à 21 12 58

#### In French, while inputing text
Capture d’écran 2020-08-31 à 21 11 52

The bug is occurring on Firefox / Safari / Chrome

## How to reproduce

Access https://nuxtjs.org and try to use the search function

## Expected display

Capture d’écran 2020-08-31 à 21 30 04

### PR DESCRIPTION
**Pull request by oiagorodrigues**
Closes #624

### CODE REVIEW

File: assets/css/rtl.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Algolia search
-.algolia-wrapper {
+.algolia-wrapper.rtl {
```
Comment (lecoueyl) [inline assets/css/rtl.scss:2]: You can directly use the setting from the HTML tag
```suggestion
html[dir='rtl'] .algolia-wrapper {
```

File: components/partials/AlgoliaSearchInput.vue [hunk 2]
```diff
@@ -1,7 +1,12 @@
 <template>
   <div
     class="algolia-wrapper relative"
-    :class="[focused ? 'focused' : 'blurred', !q && 'blurred', appearance]"
+    :class="[
```
Comment (lecoueyl) [inline components/partials/AlgoliaSearchInput.vue:4]: If `html[dir='ltr']` is used to target the CSS class, no need for dynamic class to handle language direction

**Review by lecoueyl — CHANGES_REQUESTED** [review]

File: assets/css/rtl.scss (hunk 1 above)
Comment (oiagorodrigues) [reply assets/css/rtl.scss:2]: I'll apply that. Didn't know about this feature

**Review by lecoueyl — APPROVED** [review]

**Review by pi0 — APPROVED** [review]

**Comment by pi0** [comment]
Thanks!

**Pull request merged**